app/route_rangers_api/utils/heatmap_data_prep.py

```diff
@@ -0,0 +1,218 @@
+        "percentage_bus_to_work",
+        "percentage_subway_to_work",
+        "total_weighted_commute_time",
+    ]
+
+    demographics_df.loc[:, columns_to_round] = demographics_df.loc[
+        :, columns_to_round
+    ].round(2)
+
+    return demographics_df
+
+
+def drop_columns(demographics_df: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
+    columns_to_drop = [
+        "work_commute_time_15_29",
+        "work_commute_time_30_44",
+        "work_commute_time_45_59",
+        "work_commute_time_60_89",
+        "work_commute_time_less_15",
+        "work_commute_time_over_90",
+        "transportation_to_work_public",
+        "transportation_to_work_bus",
+        "transportation_to_work_subway",
+        "weighted_commute_15_29",
+        "weighted_commute_30_44",
+        "weighted_commute_45_59",
+        "weighted_commute_60_89",
+        "weighted_commute_less_15",
+        "weighted_commute_over_90",
+    ]
+    return demographics_df.drop(columns=columns_to_drop)
+
+
+def simplify_geojson(
+    gdf: gpd.GeoDataFrame, tolerance: float = 0.001
+) -> gpd.GeoDataFrame:
+    """
+    Simplify the geometries in the 'geographic_delimitation'
+    column to help with loading speed
+    """
+    gdf.loc[:, "geographic_delimitation"] = gdf["geographic_delimitation"].apply(
+        lambda geom: geom.simplify(tolerance, preserve_topology=True)
+    )
+    return gdf
+
+
+def main() -> None:
+    setup_django()
+    # configuration to save output
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    parent_dir = os.path.abspath(os.path.join(current_dir, "../../"))
+    sys.path.append(parent_dir)
+    static_folder = os.path.join(parent_dir, "route_rangers_api/static")
+    chicago_merged_path = os.path.join(static_folder, "ChicagoCensus_2020.geojson")
+    newyork_merged_path = os.path.join(static_folder, "NewYorkCensus_2020.geojson")
+    portland_merged_path = os.path.join(static_folder, "PortlandCensus_2020.geojson")
+
+    # call demographic data functions
+    demographics_df = fetch_demographics_data()
+    demographics_df = preprocess_demographics_df(demographics_df)
+    demographics_df = calculate_demographic_metrics(demographics_df)
+    demographics_df = drop_columns(demographics_df)
+
+    demographics_gdf = gpd.GeoDataFrame(
+        demographics_df, geometry="geographic_delimitation", crs="EPSG:4326"
+    )
+
+    # split gdfs by city and apply geo simplification
+    chicago_gdf = simplify_geojson(demographics_gdf[demographics_gdf["state"] == "17"])
+    newyork_gdf = simplify_geojson(demographics_gdf[demographics_gdf["state"] == "36"])
+    portland_gdf = simplify_geojson(demographics_gdf[demographics_gdf["state"] == "41"])
+
+    # Save geojsons for visualizing heat map
+    chicago_gdf.to_file(chicago_merged_path, driver="GeoJSON")
+    newyork_gdf.to_file(newyork_merged_path, driver="GeoJSON")
+    portland_gdf.to_file(portland_merged_path, driver="GeoJSON")
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
sorry for the confusion on putting this in `utils/`, I thought this script would dynamically query the database and then get pulled into `views.py` but if I am reading this correctly it is more of an ingestion/data processing step that writes out a file and the frontend grabs the file? 

Since that is the case and you run it from the command line (which was why you had all those tough filepath reconfigs because they) I think this should be moved to the `scripts/` folder so then it will automatically have access to the settings, then:
- [ ] remove path reworking for settings 
- [ ] change `main` to `run` like the ingestion scripts do 
- [ ] test running it with `python -m manage runscript heatmap_data_prep`